stacks-common/src/util/versions.rs

```diff
@@ -0,0 +1,82 @@
+use serde::Serialize;
+use serde::Deserialize;
+use std::fs::File;
+
+#[derive(Serialize, Deserialize)]
+struct Versions {
+    stacks_node: String,
+    stacks_signer: String,
+    blockstack_cli: String,
+    clarity_cli: String,
+    relay_server: String,
+    stacks_inspect: String,
+}
+
+pub fn get_build_version(binary: &str) -> String {
+    let versions = get_build_versions();
+    match binary {
+        "stacks-node" => versions.stacks_node,
+        "stacks-signer" => versions.stacks_signer,
+        "blockstack-cli" => versions.blockstack_cli,
+        "clarity-cli" => versions.clarity_cli,
+        "relay-server" => versions.relay_server,
+        "stacks-inspect" => versions.stacks_inspect,
+        _ => panic!("Binary not found"),
+    }
+}
+
+pub fn get_long_version(binary: &str) -> String {
+    let build_version = get_build_version(binary);
+    format!("{} ({}, {}, {})", build_version, get_target_build_type(), get_target_os(), get_target_arch())
+}
```

Comment:
We already have the `version_string` util, which includes the arch, build type, and also git info. I think we should keep that, as it's also a part of our CI process (ie setting the git info via ENV).